Signal/Images.xcassets/mute-selected-wide.imageset/Contents.json

```diff
@@ -2,7 +2,7 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "video-active.png",
+      "filename" : "mute-selected-wide.png",
```

Comment:
I know you want a different style for the "unselected" icons, but I'm trying to prioritize getting things working first.  I'll return to these icons if we have time.